npm-shrinkwrap.json

```diff
@@ -9211,6 +9211,14 @@
           "version": "1.1.0"
         }
       }
+    },
+    "source-map": {
```

Comment:
I just noticed that is slightly different to the previous one: https://github.com/Automattic/wp-calypso/pull/3849/files#diff-36018a6a9e1de1798516799b84841318L6762